src/driver/experimental_trial.py

```diff
@@ -114,6 +114,9 @@ def evaluate(self, rng: np.random.Generator) -> pd.DataFrame:
         logging.vlog(2, f"Dataset {self._data_set_name}")
         logging.vlog(2, f"Trial   {self._trial_descriptor}")
 
+        rng = np.random.default_rng(seed=seed)
+        np.random.seed(seed)
```

Comment:
Could you explain the function of this line in addition to the above line?

Reply:
In earlier distributions of numpy, random numbers were generated via RandomState.  This was subsequently replaced with Random Generator.  The Planning Evaluation Framework uses the more modern Random Generator throughout.  The seed for this generator is initialized at line 117.  However, some of the code that we depend on (specifically, PyDOE.lhs) uses the legacy RandomState.  The seed for this generator is initialized at line 118.

For more information, see the following:
https://numpy.org/doc/stable/reference/random/generator.html